documents/motions/10.2020.7 - Ethics Committee.md

```diff
@@ -28,5 +28,5 @@ The WCA Ethics Committee is an Advisory Committee of the WCA.
       2. Requests for appeal by the WCA Board on behalf of a Registered Speedcuber or WCA Staff.
       3. Financial Support of the WCA Ethics Committee by the WCA Financial Committee.
       4. Questions and reports to the confidant within the WCA Ethics Committee.
-2. All WCA Staff must fully cooperate with investigations by the WCA Ethics Committee.
+2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Ethics Committee.
```

Comment:
I am not sure this is correct. It feels like being forced to dig your own grave. For staff it is different because they are enployees and may choose to leave the organisation.

Reply:
Since WEC only deal with Staff incidents, this shouldn't be the case that Registered Speedcubers are "digging their own grave" because WEC wouldn't be investigating them. One thing to note, is that if WEC becomes non-functioning, then WDC would take over their role. If the WDC became non-functioning it would make sense to have WEC perform their role, therefore this clause would make sense.

@Epride maybe the WDC / WEC motions should be updated to show who is taking on WDC responsibilities in case of WDC being non-functioning. I would assume it's WEC, but I can see the case being made for Board as well.

Reply:
This serves two purposes:
1. Usually WEC investigates Staff conduct, so in most cases a Registered Speedcuber would be a witness, not the person being investigated.
2. In cases in which the WEC is investigating a WDC matter (due to a conflict of interest, for example), this is already a requirement for WDC investigations:

> 3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary
> Committee.

This just eliminates the discrepancy of having the process handled in two different ways depending on who conducts the investigation.